style.css

```diff
@@ -180,4 +476,260 @@
+    padding: 16px;
+    gap: 12px;
+    width: 295px;
+    margin-left: 22px;
+    background: #fff;
+  }
+
+  .section-nav .header-mobile {
+    height: 220px;
+  }
+
+  .section-mobile {
+    display: grid;
+    grid-template-columns: auto min-max0(0, 1fr);
+    grid-template-rows: auto 1fr;
+    width: 295px;
+    row-gap: 12px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .header-mobile img {
+    width: 295px;
+  }
+
+  .section-mobile .project-title h2 {
+    font-family: 'Poppins', serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+    margin: 0;
+  }
+
+  .section-mobile .project-title {
+    display: flex;
+    flex-direction: column;
+    height: 92px;
+  }
+
+  .section-mobile .project-title .top-list {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .section-mobile .list-bottom li {
+    font-family: 'Poppins', serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    color: #6070ff;
+    background: #ebebff;
+    border-radius: 8px;
+    padding: 12px 12px 24px 12px;
+  }
+
+  .section-mobile .project-title .top-list li {
+    font-family: 'Poppins', serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  .about-myself .about-container .language li {
+    display: inline;
+    list-style: none;
+  }
+
+  .about-myself .about-container .language-list .list {
+    display: flex;
+    gap: 16px;
+    align-items: center;
+    background: #f7f7f9;
+    border-radius: 8px;
+    width: 303px;
+    height: 66.72px;
+    padding-left: 12px;
+  }
+
+  .about-myself .about-container .social-media ul li {
+    display: inline;
+    list-style: none;
+    height: 20px;
+  }
+
+  .section-mobile .project-title .top-list li:first-child {
+    color: #344563;
+  }
+
+  .section-mobile p {
+    font-family: 'Poppins', serif;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 24px;
+    vertical-align: top;
+    color: #344563;
+    margin: 0;
+  }
+
+  .section-mobile .list-bottom {
+    display: flex;
+    gap: 8px;
+    list-style: none;
+  }
+
+  .section-mobile .action {
+    display: flex;
+    align-items: center;
+  }
+
+  .section-mobile .button {
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    outline: none;
+    text-align: center;
+    letter-spacing: 0.03em;
+    padding: 16px 16px 40px 16px;
+    width: 124px;
+    height: 48px;
+    font-family: 'Poppins', serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 24px;
+    color: #396df2;
+    background: #fff;
+    cursor: pointer;
+  }
+
+  .section-mobile .button:hover {
+    background: #6070ff !important;
+    color: #fff;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  }
+
+  .section-mobile .button:active {
+    color: #fff;
+    background: #2230d2;
+  }
```

Comment:
[OPTIONAL] Taking a look at your project using Google Chrome, this style is not being applied to your buttons in the `Works` section. Consider fixing this as it is a project requirement.

<img width="834" alt="Screen Shot 2022-05-30 at 14 22 40" src="https://user-images.githubusercontent.com/82426682/171050216-92d5040a-4363-45b1-96cd-87cfe035323a.png">